### PR TITLE
1161746 alerts

### PR DESCRIPTION
**Pull request by willkg**
r?

### CODE REVIEW

**Comment by willkg** [comment]
The tests are pretty comprehensive. As long as they look good, this is probably fine.

File: fjord/alerts/api_views.py [hunk 1]
```diff
@@ -65,6 +62,46 @@ def has_object_permission(self, request, view, obj):
...
+            raise serializers.ValidationError(
+                'start_time_start must occur before start_time_end.')
+
+        if is_after(data.get('end_time_start'), data.get('end_time_end')):
+            raise serializers.ValidationError(
+                'end_time_start must occur before end_time_end.')
+
+        if is_after(data.get('created_start'), data.get('created_end')):
+            raise serializers.ValidationError(
+                'created_start must occur before created_end.')
```
Comment (rlr) [inline fjord/alerts/api_views.py]: I didn't see these validations in the tests.

**Comment by rlr** [comment]
Looks good. Pretty thorough coverage minus the is_after stuff which is fine but you were already being thorough enough I figured would be nice to have those too.

File: fjord/alerts/api_views.py (hunk 1 above)
Comment (willkg) [inline fjord/alerts/api_views.py]: Oops--you're right. I should add that.

**Comment by willkg** [comment]
^^^ That adds docs and the missing tests.

**Comment by willkg** [comment]
^^^ Added some more examples to the docs covering Cheng's use cases.

**Comment by rlr** [comment]
Awesome, LGTM. :ship: r+

**Comment by willkg** [comment]
Thank you!

**Pull request merged**